corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/utils.js

```diff
@@ -134,6 +134,7 @@ hqDefine("cloudcare/js/formplayer/menus/utils", function () {
             sortIndices: menuResponse.sortIndices,
             isMultiSelect: menuResponse.multiSelect,
             multiSelectMaxSelectValue: menuResponse.maxSelectValue,
+            hasDynamicSearch: menuResponse.hasDynamicSearch,
```

Comment:
Is there any risk if this has the possibility to be `undefined` (which I think it does, since it only exists on the `queryResponse`)?